install/kubernetes/cilium/README.md

```diff
@@ -159,6 +159,7 @@ contributors across the globe, there is almost always someone available to help.
 | hostServices.enabled | bool | `false` | Enable host reachable services. |
 | hostServices.protocols | string | `"tcp,udp"` | Supported list of protocols to apply ClusterIP translation to. |
 | hubble.enabled | bool | `true` | Enable Hubble (true by default). |
+| hubble.listenAddress | string | `":4244"` | An additional address for Hubble to listen to. Set this field ":4244" if you are enabling Hubble Relay, as it assumes that Hubble is listening on port 4244. |
```

Comment:
not directly related to this PR, but i guess relay can now use `hubble.listenAddress` to figure out which port hubble is listening on instead of assuming 4244 cc @Rolinh

Reply:
Hubble Relay is relying solely on the peer service and making it rely on other sources could create inconsistencies.
IIRC, the peer service already users this value and assume it's the same one for every peer (I'd need to check to be sure, it could also just be using the default value for the port in the code).

Reply:
@Rolinh The peer service doesn't provide the port in the ChangeNotification, only the IP address: https://github.com/cilium/cilium/blob/2e3b9e8174ea511dd9d16ff263e1bb6d31fcfff0/pkg/hubble/peer/handler.go#L142-L154
Then relay assume the default port (hardcoded constant):
https://github.com/cilium/cilium/blob/45b5abf10a9814bbaa6b200968eee1872a18433f/pkg/hubble/peer/types/peer.go#L87

Reply:
@kAworu Ah right, I remember now. I think this information should be added to the peer service. IIRC, Relay uses the default value for the port if the peer service does not provide this information already.

Reply:
@rochacon agreed, [we already have an issue for that](https://github.com/cilium/cilium/issues/13034) FWIW.